app/services/user/update.rb

```diff
@@ -0,0 +1,20 @@
+class User
+  class Update
+    def initialize(user:, params:)
+      @user = user
+      @params = params
+    end
+
+    def call
+      if params[:password].blank? && params[:password_confirmation].blank?
```

Comment:
That's a matter of personal taste but I believe that using `params` inside of service objects looks wrong. 
I think that something like this would be much nicer in terms or readability & flexibility
```ruby
def initialize(params)
   @user     = params[:user]
   @password = params[:password]
   @email    = params[:email]
    ....
end

private

attr_reader :email, :user, :password
```
1. Your service isn't filled with sequential `params` calls which is more readable
2. It's easy to swap `attr_reader` implementations once there's a need, meaning it's more flexible
3. Yep, you'll have to implement `user_params` method to use with `update` call but I believe it's much more tangible due to the fact that you don't have to look at the controller to see which values are updated & you've got it all in one place, meaning it's easy to  change once the need arises

Anyways, that's just my opinion so decide for yourself, peace :vulcan_salute: